apps/ucp-ui/src/shared/constants/environment.test.ts

```diff
@@ -1,23 +1,23 @@
 describe("Environment based urls", () => {
   beforeEach(() => {
-    delete process.env.UCP_UI_REVIEW_APP;
+    delete process.env.REVIEW_APP;
```

Comment:
I decided to refactor and simplify this name because although it only applies to ucp-ui currently, it could be applied to any service. Also, because build is now checking for Review, staging, and production I thought removing the `UCP_UI` from the name simplifies things and makes it more readable.